docs/contributing.md

```diff
@@ -0,0 +1,215 @@
+---
+id: contributing
+title: Contributing to the Doc Hub
+slug: /contributing
+---
+
+The Experimenter documentation hub is managed in the [mozilla/experimenter-docs](https://github.com/mozilla/experimenter-docs) repository. You will need a GitHub account to modify an existing document or create a new one.
+
+This page will walk you through how to edit an existing document or create a new one, how to display docs in the sidebar where desired, and how to make these changes in GitHub.
+
+Documents are written in Markdown (`.md`) files using [GitHub-flavored Markdown syntax](https://github.github.com/gfm/) and are compiled into static web pages. [MDX](https://mdxjs.com/) is also supported, meaning you may embed React components in content as needed. Additionally, Docusaurus offers [admonitions](https://v2.docusaurus.io/docs/markdown-features/admonitions) which can be useful when placing emphasis on pieces of documentation. Refer to Docusaurus [Markdown features](https://v2.docusaurus.io/docs/markdown-features) for more technical information about these Markdown pages.
+
+<div className="flex-lg">
+<div>
+
+**Notifications**
+
+If you would like to subscribe to notifications for this repository, including for when someone requests doc changes with a pull request that you can potentially review and approve, be sure to "Watch" the repository.
+
+Also, see [configuring notifications](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github/configuring-notifications) in GitHub.
+
+Alternatively, consider subscribing to this repository in the [GitHub Slack](https://slack.github.com/) app by searching for it under "Apps" inside Slack.
+
+</div>
+   <img src="../img/github-watch.png" alt="GitHub Watch" className="img-xs-right" />
```

Comment:
Maybe this could be more descriptive? Like "From the repository's 'Notifications settings' menu, select 'All activity' to be notified of all changes to this repository"